app/models/campaign.rb

```diff
@@ -55,6 +55,46 @@ def last_contribution
     ).order(created_at: :desc).first&.created_at
   end
 
+  def seller_distributor_pairs
```

Comment:
does anyone know a better way to do this in rails? i'm trying to do something in 1 query, but i couldn't find a way... i'd need something like:

```
    CampaignsSellersDistributor
      .find_by_sql(
        [
          """
          SELECT
            csd.id,
            csd.campaign_id,
            csd.seller_id,
            st.name as seller_name,
            s.hero_image_url as seller_image,
            csd.distributor_id,
            d.name as distributor_name,
            d.image_url as distributor_image
          FROM campaigns_sellers_distributors csd
          JOIN campaigns c ON csd.campaign_id=c.id
          JOIN sellers s ON csd.seller_id=s.id
          JOIN seller_translations st ON st.seller_id=csd.seller_id
          JOIN distributors d ON csd.distributor_id=d.id
          WHERE csd.campaign_id=? AND st.locale=?
          """,
          id,
          ???,
        ]
      )
```

Reply:
also please comment on my ruby code i don't know any ruby so i'm pretty sure this code is trash